### ISSUE TITLE
Modifying depth_evaluation_utils for a custom dataset

### ISSUE THREAD

**Issue by starkgate** [guangmingw/DOPlearning#3]
I'd like to test the pretrained networks you provide on a custom, 82-frame dataset with just no scene changes. I looked at the code in `stillbox_eval/depth_evaluation_utils.py`, however much remains unclear. For instance:
- What is the code's understanding of `scene`, `ref_imgs` and `tgt`?
- Is the first frame of my sequence as the tgt and the rest sequentially listed in ref_imgs?
- What if I don't have any depth map to provide? I just want to test the pretrained dataset, not train the network myself

In short, could you provide a simple overview of the steps needed to load a custom dataset?

**Comment by starkgate** [comment]
Alright, I think I understand it now. I'll put my code and explain it here if anyone else has the same problem. Please correct me if I'm wrong! `ref_imgs` are the frames around the current frame (eg at frame t, frames t-1 and t+1).
```
def read_scene_data(data_root, test_list, seq_length=3, step=1):
    config = configparser.ConfigParser()
    config.read(os.path.join(data_root, 'seqinfo.ini'))

    im_files = []
    # how many frames around the current (tgt) frame should be taken into account by the network
    demi_length = (seq_length - 1) // 2
    # by default 1 frame before and 1 after: [-1, 1]
    shift_range = [step*i for i in list(range(-demi_length, 0)) + list(range(1, demi_length + 1))]

    for index, sample in enumerate(tqdm(test_list)):
        if os.path.isfile(sample):
            # clamp indices between 1 and the maximum number of frames in the scene
            capped_indices_range = list(map(lambda x: min(max(0, index + x), int(config['Sequence']['seqLength']) - 1), shift_range))
            ref_imgs_path = [test_list[ref_index] for ref_index in capped_indices_range]
            im_files.append([sample, ref_imgs_path])
        else:
            print('{} missing'.format(sample))

    return None, im_files, None
```
